### PR TITLE
Fix 2036 Export and import for missing data agent

### LINKED ISSUE #2036: Export-import for missing data agent
Expand the missing data agent implemented in #1931 to include export-import and local storage functionality.
* After selecting a set of object/properties, the user should be able to export the set, possibly as a .json file. It would be good to have a modal pop up where the user can enter the file name.
* Importing a valid file should remove any existing rows from the missing data modal, and fill it based on the file contents.
* After the user makes any changes in the missing data modal, the rows should be saved to local storage. The rows should be retrieved from local storage if the user opens the missing data modal at a later time.
* A "Clear" button should be implemented to remove all rows from the missing data modal.

### PR DESCRIPTION
**Pull request by eliasbruvik**
[//]: # (Request Template Witsml Explorer)

[//]: # (Thank you for contributing to WITSML Explorer! Before submitting this PR, please fill in the following template.)

## Fixes

[//]: # (Write the GitHub issue number starting with #, or, for Equinor only, the Jira issue number starting with WE-)

This pull request fixes #2036 

## Description

[//]: # (Please include a written summary of the changes and which issue is fixed. List any dependencies that are required for this change._)

Added functionality to export, import and store missing data agent checks in localstorage.

## Type of change

[//]: # (Mark any of the types of change that apply.)

* [ ] Bugfix
* [ ] New feature (non-breaking change which adds functionality)
* [x] Enhancement of existing functionality
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Impacted Areas in Application

[//]: # (List general components of the application that this PR will affect)

* [x] Frontend
* [ ] API
* [ ] WITSML
* [ ] Other (please describe)

## Checklist:

[//]: # (Please tick all the boxes or remove the ones that aren't needed and explain why)

*Communication*
* [ ] I have made corresponding changes to the documentation
* [ ] PR affects application security

*Code quality*
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [ ] New code is covered by passing tests

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by eliasbruvik** [comment]
Blocked until #2033 is merged

**Review by janmarius — APPROVED** [review]
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code

**Pull request merged**
